<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="All Test Suite" duration-ms="78506" started-at="2022-05-18T19:05:53Z" finished-at="2022-05-18T19:07:12Z">
    <groups>
    </groups>
    <test name="LoginTest" duration-ms="37793" started-at="2022-05-18T19:05:53Z" finished-at="2022-05-18T19:06:31Z">
      <class name="test.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.LoginTest@45d84a20]" name="setUp" is-config="true" duration-ms="45" started-at="2022-05-18T16:05:53Z" finished-at="2022-05-18T16:05:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:test.LoginTest@45d84a20]" name="login" duration-ms="30327" started-at="2022-05-18T16:05:53Z" data-provider="dataLogin" finished-at="2022-05-18T16:06:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[nonrestricteduser@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[restricted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:test.LoginTest@45d84a20]" name="login" duration-ms="5620" started-at="2022-05-18T16:06:23Z" data-provider="dataLogin" finished-at="2022-05-18T16:06:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[restricteduser@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[45678]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[restrcted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.LoginTest@45d84a20]" name="tearDown" is-config="true" duration-ms="1790" started-at="2022-05-18T16:06:29Z" finished-at="2022-05-18T16:06:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.LoginTest -->
    </test> <!-- LoginTest -->
    <test name="ValidateTitle" duration-ms="23722" started-at="2022-05-18T19:06:31Z" finished-at="2022-05-18T19:06:55Z">
      <class name="test.ValidateTitleTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.ValidateTitleTest@6cb107fd]" name="setUp" is-config="true" duration-ms="23389" started-at="2022-05-18T16:06:31Z" finished-at="2022-05-18T16:06:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="validarLogin()[pri:0, instance:test.ValidateTitleTest@6cb107fd]" name="validarLogin" duration-ms="25" started-at="2022-05-18T16:06:54Z" finished-at="2022-05-18T16:06:54Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED COURSES1] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED COURSES1] but found [FEATURED COURSES]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at test.ValidateTitleTest.validarLogin(ValidateTitleTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validarLogin -->
      </class> <!-- test.ValidateTitleTest -->
    </test> <!-- ValidateTitle -->
    <test name="ValidateNavigation" duration-ms="16780" started-at="2022-05-18T19:06:55Z" finished-at="2022-05-18T19:07:12Z">
      <class name="test.ValidateNavigation">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.ValidateNavigation@7bd7d6d6]" name="setUp" is-config="true" duration-ms="16741" started-at="2022-05-18T16:06:55Z" finished-at="2022-05-18T16:07:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validarLogin()[pri:0, instance:test.ValidateNavigation@7bd7d6d6]" name="validarLogin" duration-ms="39" started-at="2022-05-18T16:07:12Z" finished-at="2022-05-18T16:07:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validarLogin -->
      </class> <!-- test.ValidateNavigation -->
    </test> <!-- ValidateNavigation -->
  </suite> <!-- All Test Suite -->
</testng-results>
